.gitpod.yml

```diff
@@ -1,28 +1,39 @@
 image: mortenson/tome
 tasks:
-  - name: Install tome
-    command: |
-      COMPOSER_MEMORY_LIMIT=2048M composer install
-      [ -d content ] && drush tome:install -y
-      gp sync-done install
   - name: Init and login
-    init: |
-      echo "Waiting for install..."
-      gp sync-await install
     command: |
       eval $(gp env -e DRUSH_OPTIONS_URI="0.0.0.0:8888")
       [ ! -d content ] && echo "Looks like you haven't initialized a Tome site yet - make sure to composer require any profiles you want to use (ex: 'composer require drupal-tome/bookish'), then run drush tome:init to get started!"
       [ -d content ] && drush uli
   - name: Runserver
     init: |
-      echo "Waiting for install..."
-      gp sync-await install
+      COMPOSER_MEMORY_LIMIT=2048M composer install
+      [ -d content ] && drush tome:install -y
     command: |
       [ ! -d content ] && echo "When you're finished running drush tome:init you can use this tab to run your server with 'drush runserver 0.0.0.0:8888'"
       [ -d content ] && drush runserver 0.0.0.0:8888
     openMode: split-right
 ports:
   - port: 8888
-    onOpen: ignore
+    onOpen: open-preview
   - port: 8889
     onOpen: ignore
+
+github:
+  prebuilds:
+    # enable for the master/default branch (defaults to true)
+    master: true
+    # enable for all branches in this repo (defaults to false)
+    branches: true
+    # enable for pull requests coming from this repo (defaults to true)
+    pullRequests: true
+    # enable for pull requests coming from forks (defaults to false)
+    pullRequestsFromForks: true
```

Comment:
Are there security implications to this for public repos? There are with GitHub actions which is why I'm wondering

Reply:
I saw a discussion that `pullRequestsFromForks: true` might expose variables in certain settings, so perhaps that one should be set to `false` ?